ons_ras_common/ons_asyncio.py

```diff
@@ -0,0 +1,139 @@
+##############################################################################
+#                                                                            #
+#   Generic ASYNC IO Routines for accessing remote REST endpoints            #
+#   License: MIT                                                             #
+#   Copyright (c) 2017 Crown Copyright (Office for National Statistics)      #
+#                                                                            #
+##############################################################################
+#
+#   ONSAsyncIO wraps generic endpoint access routines
+#
+##############################################################################
+import crochet
+import treq
+from json import loads, dumps
+from twisted.internet import defer
+#
+#   How long (by default) do we wait for an endpoint call before we timeout?
+#
+DEFAULT_TIMEOUT = 3
+
+
+class ONSAsyncIO(object):
+    """
+    This class is designed to take all the work out of accessing the case service. Initially it
```

Comment:
Is this a correct description of what this class does, as it looks like a copy from ONSCase? I'm unclear on what this class is for or why we need it.